changelog/front-commerce-3.0.mdx

```diff
@@ -0,0 +1,249 @@
+bringing all the goodness we developed over the past 7 years, to the Remix
+ecosystem.
+
+Let's examine how we've merged both technologies to enhance the experience for
+both end users and developers.
+
+<!--truncate-->
+
+import ChangelogFooter from "@site/src/components/ChangelogFooter";
+import BackportList from "@site/src/components/BackportList";
+import ContactLink from "@site/src/components/ContactLink";
+
+## 💌 A team and community effort
+
+As we embark on the exciting journey of this new major product release, I can't
+help but feel a deep sense of gratitude. A lot happened since
+[we announced that we were Remixing Front-Commerce 6 months ago](/changelog/front-commerce-is-getting-a-remix)
+and we want to say a few words to our community.
+
+First and foremost, a heartfelt thanks goes out to our incredible team who
+poured their passion, creativity, and countless hours into bringing this vision
+to life. Without their dedication, none of this would have been possible.
+
+To our invaluable integrators (agencies, developers, project managers…), who
+have been providing feedback over the years and kept interacting with us while
+we were busy with this new version: your understanding and collaboration mean
+the world to us, and we can't wait for you to experience the magic of a
+Front-Commerce Remixed! **We've built it for you!**
+
+And for merchants trusting Front-Commerce, we genuinely thank you for your
+patience as we focused on v3 development over summer features. We're eagerly
+anticipating how this new release will enhance your business.
+
+Together, we've created something special: let's dive into what this release
+holds and how it will positively impact all of you. ☕
+
+## Web Performance Improvement
+
+We're all working together to create user experiences that convert, fast! Let's
+take a moment to explore the improvements that will led to faster page load
+times and enhanced user experiences.
+
+### Reduced Client-side Dependencies
+
+With the introduction of Front-Commerce 3.0, we've made significant strides in
+reducing client-side dependencies. We've removed libraries like Apollo Client,
+GraphQL, GraphQL-tag, Formsy, and RxJS, replacing them with Remix default data
+fetching and primitives.
+
+This translates to **less JavaScript shipped to users**, ensuring that your
+application is faster to interact with, particularly on mobile and low-end
+devices.
+
+**We could achieve this thanks to a major shift in the way data flows in
+Front-Commerce**: instead of using GraphQL over HTTP (2.x), Front-Commerce now
+uses GraphQL server-side and uses Remix's default fetching patterns (`loaders` /
+`actions` / `fetchers`) for client-server interactions. Read
+[Server vs. Client Code Execution (Remix doc)](https://remix.run/docs/en/main/discussion/server-vs-client)
+to learn about this.
+
+### Streaming Responses to Users
+
+This new version, improves how content is delivered to your users. Unlike
```

Comment:
**thought:** this comma feels a bit weird here, splitting the sentence that is already not that long.

```suggestion
This new version improves how content is delivered to your users. Unlike
```